recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -350,7 +350,7 @@ fi
 
 title "Creating tarball"
 FILE="/tmp/support-data_$(date "+%Y%m%d-%H%M%S").tgz"
-tar czf "$FILE" -C "$TMPDIR" .
+tar czhf "$FILE" -C "$TMPDIR" .
```

Comment:
This may fail with relative links if the referenced file does not happen to be in the expected location in TMPDIR. You could s/cp -a/cp -aL/ in function copy_path.